lc-hax/Scripts/Modules/StunMod.cs

```diff
@@ -16,13 +18,20 @@ public sealed class StunMod : MonoBehaviour, IStun {
         if (!Helper.CurrentCamera.IsNotNull(out Camera camera)) return;
 
         this.Stun(camera.transform.position, 5.0f);
+        int colliders = Physics.OverlapSphereNonAlloc(camera.transform.position, 5.0f, this.Colliders);
+
+        for (int i = 0; i < colliders; i++) {
+            GameObject gameObject = this.Colliders[i].gameObject;
 
-        foreach (RaycastHit raycastHit in Helper.RaycastForward()) {
-            GameObject gameObject = raycastHit.collider.gameObject;
+            if (gameObject.TryGetComponent(out Turret _) && gameObject.TryGetComponent(out Landmine _)) {
+                return;
+            }
 
-            if (gameObject.TryGetComponent(out Turret _) || gameObject.TryGetComponent(out Landmine _)) {
-                gameObject.GetComponent<TerminalAccessibleObject>()?.CallFunctionFromTerminal();
+            if (!gameObject.TryGetComponent(out TerminalAccessibleObject terminalAccessibleObject)) {
```

Comment:
`?.` is way cleaner than this 😂

Reply:
Yeah, but I've been traumatised by Unity's fake null problem.

Reply:
ah ok, I am not familiar with Unity's quirks, first time using it at all

Reply:
Yeah, I didn't know about it until I did this project as well. It was extremely painful to debug.